documentation/docs/contribute.md

```diff
@@ -9,35 +9,36 @@ keywords:
 - Identity Initiative
 - Documentation
 - Discord
+- reference
 ---
 
-**Thanks for thinking about contributing to the project! We have the following ways that you can contribute.**
+**Thanks for thinking about contributing to the project! You can contribute using the following methods.**
 
 ## Join the Identity Initiative
 
-The [Identity Initiative](https://github.com/iota-community/X-Team_IOTA_Identity) is a collaborative effort to help improve the developer experience.
+The [Identity Initiative](https://github.com/iota-community/X-Team_IOTA_Identity) is a collaborative effort to help improve the developer experience that includes:
 
-- Quality assurance and review
-- Documentation
-- Code samples
+- Quality assurance and review.
+- Documentation.
+- Code samples.
 
-If you'd like to get involved, join the #experience channel on [Discord](https://discord.iota.org).
+If you would like to get involved, join the #experience channel on [Discord](https://discord.iota.org).
 
-## Contribute to the project's GitHub repository
+## Contribute to the Project's GitHub Repository
 
-All the code is open source and hosted on [GitHub](https://github.com/iotaledger/identity.rs) where you can do the following:
+All of the code is open source and hosted on [GitHub](https://github.com/iotaledger/identity.rs) where you can:
 
-- Report a bug
-- Suggest a new feature
-- Contribute to the documentation
+- Report a bug.
```

Comment:
We could add direct links for these.  https://github.com/iotaledger/identity.rs/issues/new/choose

Reply:
Done